models.h

```diff
@@ -82,10 +82,7 @@ namespace Models
                   const Parameters &prm)
         {
           out << "k1=" << prm.k1 << ", "
-              << "k2=" << prm.k2 << ", "
-              << "w=" << prm.w << ", "
-              << "maxsize=" << prm.maxsize << ", "
-              << "n_variables=" << prm.n_variables;
+              << "k2=" << prm.k2 ;
```

Comment:
I guess it makes sense to only output those quantities that can be varied.

Reply:
Yes, although I would also like a way to output all parameters as well. I might put that in a member function. SampleFlow requires a << operator, right? For the MCMC output it makes sense to me to only output quantities that can be varied.

Reply:
Yes, SampleFlow requires `operator<<`, but you can always first convert to `std::valarray` and then output.